manifests/init.pp

```diff
@@ -714,6 +725,14 @@
         require => File['C:/ProgramData/Datadog'],
       }
 
+      file { 'C:/ProgramData/Datadog/install_info':
+        owner   => $dd_user,
+        group   => $dd_group,
+        mode    => '0660',
```

Comment:
Why different permissions? (640 or 660)

Reply:
I am not sure what the reason is for it but `datadog.yaml` has those permissions on Windows, so I guessed `install_info` should have the same. Should I change them on `datadog.yaml` too?

Reply:
Seems like it is used to give the administrator group access to the file (see #571)